tests/test_smoothquant/test_smoothquant_linear.py

```diff
@@ -0,0 +1,42 @@
+import warnings
+
+import pytest
+import torch
+from packaging import version
+
+try:
+    from colossalai.kernel.op_builder.smoothquant import SmoothquantBuilder
+
+    smoothquant_cuda = SmoothquantBuilder().load()
+    HAS_SMOOTHQUANT_CUDA = True
+except ImportError:
+    warnings.warn("CUDA gptq is not installed")
+    HAS_SMOOTHQUANT_CUDA = False
+
+TRITON_CUDA_SUPPORT = version.parse(torch.version.cuda) > version.parse("11.4")
+
+
+@pytest.mark.skipif(
+    not TRITON_CUDA_SUPPORT or not HAS_SMOOTHQUANT_CUDA,
```

Comment:
It does not check whether triton is properly installed

Reply:
It dose not depend on triton. Update the skip condition to only check CUDA.

Reply:
Updated.